Src/RVEngine/README.md

```
@@ -0,0 +1,84 @@
+    case DLL_THREAD_ATTACH:
+    case DLL_THREAD_DETACH:
+    case DLL_PROCESS_DETACH:
+        break;
+    }
+    return TRUE;
+}
+```
+
+## Completion Status
+
+As of now (15/02/2023) Intercept is over 97% language feature complete. You can view the progress of wrapper completion [here](https://github.com/intercept/intercept/issues/13). Almost all normally used SQF functions are available to the end user, and with the added ability of writing inline SQF code, you can call any function that has not had a wrapper written for it yet (though with a small performance penalty).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)